template/python/cyan/main.py

```
@@ -7,12 +7,13 @@
     return Cyan(
         processors=[
-{{processorsConfig}}
+# {{processorsConfig}}
         ],
         plugins=[
-{{pluginsConfig}}
+# {{pluginsConfig}}
         ],
     )
 
+print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)